buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -315,6 +316,7 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
             'include_timeseries_zone_temperatures': False,
             'include_timeseries_airflows': False,
             'include_timeseries_weather': False,
+            'timeseries_timestamp_convention': 'end',
```

Comment:
So it looks like the new YAML argument is not required and defaulted to end. This would maintain the current behavior which is good.